Empire - The Last Battle/Assets/Meshes/Banners.meta~ewan_master

```diff
@@ -0,0 +1,9 @@
+fileFormatVersion: 2
```

Comment:
Can you delete this stuff? Maybe it's backups that were added when merging but the merge wasn't completed? I don't know but it shouldn't be here ;)